assets/js/123form_submit.js

```diff
@@ -1,21 +1,28 @@
 function enable_form_submit(){
   $('.js_saver').click(function () {
-    var submitted_form = $(this).closest("form")
-    var my_destination = submitted_form.attr("action");
-    var submitted_datas = submitted_form.serialize();
-    $.ajax({type: "POST", data: submitted_datas, url: my_destination, async: true}).success(function(response){
-      var parsed_response = jQuery.parseJSON(response);
-      if(parsed_response.status == 400){
-        load_review_page(parsed_response);
-      }
-      else{
-        load_confirmation_page(parsed_response);
-      }
-    }).error(function(parsed_response){
-      load_error_page();
-    });
-    return false;
+    if (content_for_step_is_valid($(this))){
+      submit_form($(this));
+    }
+  });
+}
+
+function submit_form(clicked_button){
+  clicked_button = $('.js_saver');
+  var submitted_form = clicked_button.closest("form")
+  var my_destination = submitted_form.attr("action");
+  var submitted_datas = submitted_form.serialize();
+  $.ajax({type: "POST", data: submitted_datas, url: my_destination, async: true}).success(function(response){
+    var parsed_response = jQuery.parseJSON(response);
+    if(parsed_response.status == 400){
+      load_review_page(parsed_response);
```

Comment:
Identifier 'load_review_page' is not in camel case.<br>Identifier 'parsed_response' is not in camel case.